hq_superset/hq_domain.py

```diff
@@ -0,0 +1,48 @@
+import jinja2, os
+from flask import request, g, flash, redirect, url_for
+
+DOMAIN_REQUIRED_VIEWS = [
+	'TableModelView.list'
```

Comment:
```suggestion
    'TableModelView.list'
```
This module sometimes uses spaces to indent, but mostly tabs. Please use four-space indents.